video_xblock/static/html/studio-edit-playmedia_apikey.html

```diff
@@ -0,0 +1,19 @@
+{% load i18n %}
+
+<div class="custom-field-list">
+  <div class="custom-field-section-label">{% trans "3playmedia Key" %}</div>
+  <input
+    type="text"
+    class="field-data-control threeplaymedia-api-key"
+    id="xb-field-edit-{{field.name}}"
+    value="{{field.value|default_if_none:""}}"
+  >
+  <div class="field-elements-wrapper">
+    <button id="threeplaymedia-api-transcripts">
+      <span aria-hidden="true"></span>
+      {% trans "Connect to 3playmedia platform" %}<span class="sr"></span>
+    </button>
+  </div>
+  <div class="api-request threeplaymedia status-error"></div>
+  <div class="api-request threeplaymedia status-success"></div>
```

Comment:
This part has been changed. Now we use one message container and set appropriate `status-error/status-success` classes.